script/generate-appveyor-config.js

```diff
@@ -2,6 +2,7 @@ const fs = require('fs')
 const path = require('path')
 const YAML = require('yaml')
 
+/** @type {{'git-lfs': {version: string, files: Array<{platform: string, arch: string, name: string, checksum: string}>}, git: {packages: Array<{platform: string, arch: string, url: string, checksum: string}>} }} */
```

Comment:
This should be unnecessary as `resolveJsonModule` is enabled but it still doesn't work inside JS files for some reason. I'll poke at this later as I dig into transitioning to proper TS support.